src/components/Sidebar.tsx

```diff
@@ -0,0 +1,78 @@
+import clsx from 'clsx';
+import { FilterIcon, ChevronRightIcon } from '@heroicons/react/outline';
+import { ReactNode, Children, cloneElement } from 'react';
+import { useTranslation } from 'next-i18next';
+
+export function Sidebar(): JSX.Element {
+  return <div></div>;
+}
+
+interface SidebarControlProps {
+  open: boolean;
+  onChange: () => void;
+}
+
+function SidebarControl({ open, onChange }: SidebarControlProps): JSX.Element {
+  const { t } = useTranslation('common');
+  return (
+    <button
+      className={clsx(
+        'fixed bottom-8 left-1/2 z-20 -ml-[51px] flex flex-none flex-row justify-between rounded-full border-none bg-gray-800 py-2 px-4 text-white transition hover:border-white md:relative md:bottom-0 md:left-0 md:ml-0 md:border md:border-solid md:border-gray-900 md:bg-transparent',
```

Comment:
Just suggesting to break the line into shorter pieces.

```suggestion
        'fixed bottom-8 left-1/2 z-20 -ml-[51px] flex flex-none flex-row justify-between',
        'rounded-full border-none bg-gray-800 py-2 px-4 text-white transition',
        'hover:border-white md:relative md:bottom-0 md:left-0 md:ml-0 md:border md:border-solid md:border-gray-900 md:bg-transparent',
```